cfme/modeling/base.py

```diff
@@ -199,6 +199,21 @@ def delete_if_exists(self, *args, **kwargs):
                 raise NotImplementedError("Delete method is not implemented.")
         return False
 
+    def expected_summary_attributes(self):
+        """The method used for collecting expected title and expected breadcrumb active location
+        on Summary/ Details page of Entity.
+
+        Returns: tuple of expected title and breadcrumb active location
+        """
+
+        expected_title = "{} (Summary)".format(self.name)
+        expected_breadcrumb = (
+            expected_title
+            if self.browser.product_version < "5.11"
```

Comment:
Keep in mind the context - this has been moved from a view to an entity.

Here you want to use `self.appliance.version`, instead of trying to use the browser.